data-structures/stack.md

```diff
@@ -65,9 +65,20 @@ var Stack = function() {}
 
 Find a partner and show each other your own Stack class. Explain how the class you wrote works.
 
+Make a pair of two and ask the following questions to each-other.
+
+- What is difference between stack and simple array?
+- Which method does the stack follows?(LIFO)
```

Comment:
could you use this if it has the same meaning?:
"Which method uses LIFO?"

Reply:
Yes, that's a nice suggestion. 🎉 I will make it.